docs/conf.py

```diff
@@ -147,13 +149,33 @@ def metadata_to_rst(app):
     package.to_rst(path=DOCS_DIR / "data_dictionaries/pudl_db.rst")
 
 
-def cleanup_rst(app, exception):
+def static_dfs_to_rst(app):
+    """Export static code labeling dataframes to RST for inclusion in the documentation."""
+    # Sphinx csv-table directive wants an absolute path relative to source directory, but pandas to_csv wants a true absolute path
+    abs_csv_dir_path = DOCS_DIR / "data_dictionaries/code_csvs"
+    abs_csv_dir_path.mkdir(parents=True, exist_ok=True)
+    rel_csv_dir_path = "/data_dictionaries/code_csvs"
+    codemetadata = CodeMetadata.from_code_ids(sorted(CODE_METADATA.keys()))
+    codemetadata.to_rst(abs_csv_dir_path=abs_csv_dir_path, rel_csv_dir_path=rel_csv_dir_path, rst_path=DOCS_DIR / "data_dictionaries/codes_and_labels.rst")
```

Comment:
Very long line. Are the linters not catching anything > 88 characters?

Reply:
There's something wrong with my pre-commit hooks maybe? Cornflakes was never catching lines that were too long but normally the pre-commit hook does. Will look into this.

Reply:
I guess if we're going to have this linter run in the pre-commit hooks we should probably turn it on in our editors / Tox too. It'll just mean there's a big backlog of long lines that need to be formatted.

Maybe we should just switch over to using `black`. All this formatting stuff is so tedious.